tf-modules/archive/api.tf

```diff
@@ -21,7 +21,7 @@ locals {
   api_redirect_uri          = "${local.api_uri}token"
   api_env_variables = {
       acquireTimeoutMillis             = var.rds_connection_timing_configuration.acquireTimeoutMillis
-      API_BASE_URL                     = local.api_uri
+      API_BASE_URL                     = local.api_uri 
```

Comment:
Really not a big deal but there's an extra space here it seems

Reply:
Ugh, I didn't have the save/remove space in one of my worktrees.   Thanks for catching that. 

d2c3782619